### PR TITLE
Configuration table syntax fix

### PR DESCRIPTION
**Pull request by nighto**
This pull request closes issue # (_put the issue number here_)

## Checklist

- [ ] All commit messages adhere to the [appropriate format](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit) in order to trigger a correct release
- [ ] All code is written in untranspiled ECMAScript (ES 2017) and is formatted using [prettier](https://prettier.io)
- [ ] Necessary unit tests are added in order to ensure correct behavior
- [ ] Documentation has been added

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ You can provide settings to a Hops application via a `"hops"` key in your `packa
 
 | Name | Type | Default | Example | Description |
 | --- | --- | --- | --- | --- |
-| `https` | `Boolean | Object` | `false` | `true` or<br/>`{ "keyFile": "./my.key", "certFile": "./my.cert" }` | Configure HTTPS support for Hops |
+| `https` | `Boolean` or `Object` | `false` | `true` or<br/>`{ "keyFile": "./my.key", "certFile": "./my.cert" }` | Configure HTTPS support for Hops |
```
Comment (herschel666) [inline DOCUMENTATION.md]: Please write it like `Boolean \| Object`, to stick with a more formal syntax.

Comment (nighto) [reply DOCUMENTATION.md]: Done :)

Comment (herschel666) [reply DOCUMENTATION.md]: No, I meant like in a single inline code element:

```
`Boolean \| Object`
```

Comment (nighto) [reply DOCUMENTATION.md]: Coolio.

**Pull request merged**